src/components/accessschedule/accessschedule.js

```diff
@@ -1,4 +1,4 @@
-define(["dialogHelper", "datetime", "emby-select", "paper-icon-button-light", "formDialogStyle"], function (dialogHelper, datetime) {
+define(["dialogHelper", "datetime", "emby-select", "paper-icon-button-light", "formDialogStyle", "globalize"], function (dialogHelper, datetime, globalize) {
```

Comment:
for example
```suggestion
define(["dialogHelper", "datetime", "globalize", "emby-select", "paper-icon-button-light", "formDialogStyle"], function (dialogHelper, datetime, globalize) {
```

Reply:
@ferferga I was misreading the warnings as errors, there are only a few issues that @MrTimscampi has linked above that can be fixed with this change :+1: no need to revert the globalize rename.